code/web/src/modules/user/Profile.js

```diff
@@ -24,23 +26,67 @@ const Profile = (props) => (
     </Helmet>
 
     {/* Top title bar */}
-    <Grid style={{ backgroundColor: grey }}>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+    <Grid style={{ backgroundColor: grey }} justifyRight={true}>
+      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '20vw', marginRight: '24%' }}>
         <H3 font="secondary">My profile</H3>
       </GridCell>
+      
+      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '9vw' }}>
+        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
+      </GridCell>
     </Grid>
 
     <Grid>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
+        <img
+          src={`${APP_URL}//images/stock/men/4.jpg`}
+          style={{ 
+            marginBottom: '2em', 
+            height: '24em', 
+            boxShadow: level1, 
+            borderRadius: '10px'
+          }}
+        /> 
+        <H4>{props.user.details.name}</H4>
```

Comment:
This idea of calling "props" in order to access different things in the "store" makes so much more sense now after the walkthrough the front end team gave.